### DISCUSSION TITLE
Can't I use an array of strings as input in the Golang library?

### DISCUSSION THREAD

**Discussion by thelicato** [projectdiscovery/httpx#717] [Q&A] [answered]
Hello, this is the first time I use ``httpx`` as a Golang library and I was wondering if I'm forced to use an **input file** or I can pass an array of strings. If this feature is missing I think it shouldn't be too hard to add it

**Comment by olearycrew** [comment, marked as answer]
Yes, you can provide a list of strings. See this example in the README.md: https://github.com/projectdiscovery/httpx#using-httpx-as-a-library
